pom.xml

```diff
@@ -289,6 +291,19 @@
                 <version>${jackson.version}</version>
             </dependency>
 
+            <!-- JMH dependencies -->
```

Comment:
We should omit these from the root pom

Reply:
If the JMH dependencies are removed from the parent pom and only included in the benchmark pom, does it make more sense to have the JMH version property in the benchmark pom or in the parent pom (with the other dependency versions)? Spoke to @rodesai offline and we both lean towards just having the version in the benchmark pom, but wanted to get @spena 's thoughts since he originally suggested the version be moved to the parent pom. What do you think?